Update docs (#3173)
foundry-rs/starknet-foundry: CHANGELOG.md

## [Unreleased]

## [0.41.0] - 2025-04-08

### Forge

#### Changed

- forge backtrace now shows if function was inlined or not

> - forge backtrace now works regardless if the function was inlined or not

Reviewer: I am not sure if I understand this point. Before this change if execution fails in an inlined function, backtrace does not work at all?

Author: I was badly named. It should be `forge backtrace now shows full backtrace regardless if the function was inlined or not`

Author: Perfect! Changed 😎

- `--template` flag to `snforge new` command that allows selecting a template for the new project. Possible values are `balance-contract` (default), `cairo-program` and `erc20-contract`

#### Changed

- forge backtrace now shows if function was inlined or not

> - forge backtrace now works regardless if the function was inlined or not

Reviewer 2: It works, but doesn't inline change the way it is displayed? Does using `inline = never` or however it was called improve the trace? If so, we should probably mention that somewhere.

Author: > Does using inline = never or however it was called improve the trace?
After this changes it is not true

> - improved display of backtrace in contract that panicked, when `panic-backtrace` is set to `true` in `Scarb.toml`. For

Reviewer 2: Maybe `as of this release` or something, so it suggests more that this was only true at the time of the release.

> - improved display of backtrace in contract that panicked, when `panic-backtrace` is set to `true` in `Scarb.toml`. For
>   now, this feature in only in `scarb nightly-2025-03-27`

Reviewer 3: ```suggestion
- improved display of backtrace in contract that panicked, when `panic-backtrace` is set to `true` in `Scarb.toml`. This feature is available since `scarb nightly-2025-03-27`
```
```suggestion
- improved display of backtrace in contract that panicked, when `panic-backtrace` is set to `true` in `Scarb.toml`. For
  now, this feature is only in `scarb nightly-2025-03-27`
```

Author: I wouldn't personally use since. As i don't see clear ordering between stable versions and nighties

>   now, this feature in only in `scarb nightly-2025-03-27`

Reviewer 3: ```suggestion
  now, this feature is only in `scarb nightly-2025-03-27`
```
